conference_instructions.md

```diff
@@ -0,0 +1,46 @@
+# Running Liberty Bikes at a Conference
+
+The following information can be used to setup the Liberty Bikes game on a standalone laptop and local wifi router for the purposes of demoing it at a conference. 
+
+
+# Hardware needed:
+
+1. Laptop with Liberty Bikes code
+2. Ethernet cable
```

Comment:
This is optional right? I assume it's for giving a direct connection between the host laptop and the router?

Reply:
I just personally like having a hardwire vs a wifi connection from a reliability / performance standpoint.  It also allows for the wired NIC to be used for the game, and your wireless NIC can also connect to the conference wifi so you can show webpages like openliberty.io, liberty bikes github, etc.

Reply:
agreed that it's nice to have, but we should still list it as optional, so people don't think it's a requirement to run the demo